src/current/v23.1/import-performance-best-practices.md

```diff
@@ -70,7 +33,7 @@ We recommend formatting your import files as `CSV`, `DELIMITED DATA`, or `AVRO`.
 Split your dump data into two files:
 
 1. A SQL file containing the table schema.
-1. A CSV file containing the table data.
+1. A CSV or AVRO file containing the table data.
```

Comment:
i've heard from @taroface TSV is popular as well.

Reply:
Adding "delimited".